pkgs/by-name/ex/exe2hex/package.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python3
+, python3Packages
+, makeWrapper
+, expect
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "exe2hex";
+  version = "1.5.2-unstable-2020-04-27";
+  format = "other";
+
+  src = fetchFromGitHub {
+    owner = "g0tmi1k";
+    repo = "exe2hex";
+    rev = "e563b353306a0f34d96150b8992f543931f907ea";
+    hash = "sha256-wriB1k45QWNCIsSb30Z3IilTGZqnc+X1+qkRrxgDxzU=";
+  };
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  propagatedBuildInputs = [
+    expect
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/{bin,share/exe2hex}
+    cp exe2hex.py $out/share/exe2hex/
+    makeWrapper ${python3}/bin/python $out/bin/exe2hex \
+      --add-flags "$out/share/exe2hex/exe2hex.py" \
+      --prefix PYTHONPATH : ${python3Packages.makePythonPath propagatedBuildInputs}
```

Comment:
```suggestion
      --prefix PYTHONPATH : "$PYTHONPATH"
```

Reply:
can remove installPhase and just do.
```nix
  postInstall = ''
    install -Dm 555 exe2hex.py $out/bin/exe2hex
  '';
```

and there are some hardcoded paths in the script which aren't going to work and the script is almost 10 years old....

Reply:
and `/usr/bin/expect` -- the one thing added to the path.

Reply:
> and `/usr/bin/expect` which needs to change

Done

Reply:
About `certutil`, should it provided by `nss` package I guess. `powershell` by `powershell` package even if the binary is `pwsh` instead of `powershell`. `winexe` comes from https://sourceforge.net/projects/winexe